AutoCollection/RequestDataHelper.ts

```diff
@@ -0,0 +1,132 @@
+///<reference path="..\Declarations\node\node.d.ts" />
+
+import http = require("http");
+import url = require("url");
+
+import ContractsModule = require("../Library/Contracts");
+import Client = require("../Library/Client");
+import Logging = require("../Library/Logging");
+import Util = require("../Library/Util");
+
+/**
+ * Helper class to read data from the requst/response objects and convert them into the telemetry contract
+ */
+class RequestDataHelper {
```

Comment:
Note - this is NOT a new class, it was extracted from Requests.ts
Note2 - the real change is _getSessionId() function - which now calls to newly created parseSessionId()